cloudify_agent/installer/operations.py

```diff
@@ -40,13 +41,10 @@ def create(cloudify_agent, installer, **_):
             ctx.logger.info('Creating Agent {0}'.format(
                 cloudify_agent['name']))
             try:
-                response = installer.runner.run_script(script_path)
-                output = response.std_out
```

Comment:
Removed the printout block because:

1. It is incomplete. `stdout` is not enough, `stderr` is needed as well.
2. The exception will be printed by the orchestrator later anyway, so no need to repeat logging.